Initial Make Auth fails

Remember to label the issue with the appropriate area, and also assign it to yourself if you plan to work on the issue in the near future.

Following these instructions 
https://microsoft.github.io/AzureTRE/tre-admins/setup-instructions/pre-deployment-steps/

**Describe the bug**
Receiving the following output from when using make auth

Checking for Azure CLI...
»»» 🔨 Azure details from logged on user 
»»»   • Subscription: *** (value removed)
»»»   • Tenant:       *** (value removed) 

/workspaces/AzureTRE//devops/scripts/load_env.sh: line 10: export: ``make': not a valid identifier
/workspaces/AzureTRE//devops/scripts/load_env.sh: line 10: export: `auth`>': not a valid identifier
/workspaces/AzureTRE//devops/scripts/load_env.sh: line 10: export: ``make': not a valid identifier
/workspaces/AzureTRE//devops/scripts/load_env.sh: line 10: export: `auth`>': not a valid identifier
/workspaces/AzureTRE//devops/scripts/load_env.sh: line 10: export: ``make': not a valid identifier
/workspaces/AzureTRE//devops/scripts/load_env.sh: line 10: export: `auth`>': not a valid identifier
make: *** [Makefile:432: auth] Error 1
vscode ➜ /workspaces/AzureTRE (deploy ✗) $ 

**Steps to reproduce**

1. make auth from VS Code CLI terminal 
1.
1.

--- a/templates/core/.env.sample
+++ b/templates/core/.env.sample
@@ -15,10 +15,11 @@ AAD_TENANT_ID=__CHANGE_ME__
 # APPLICATION_ADMIN_CLIENT_ID=<__CHANGE_ME__Generated when you run `make auth`>
 # APPLICATION_ADMIN_CLIENT_SECRET=<__CHANGE_ME__Generated when you run `make auth`>
 
-
-API_CLIENT_ID=<__CHANGE_ME__Generated when you run `make auth`>
-API_CLIENT_SECRET=<__CHANGE_ME__Generated when you run `make auth`>
-SWAGGER_UI_CLIENT_ID=<__CHANGE_ME__Generated when you run `make auth`>
+# These values will be created when you run `make auth`. You can replace these lines.
+# They are mandatory for the running of `make all`
+# API_CLIENT_ID=<__CHANGE_ME__Generated when you run `make auth`>
+# API_CLIENT_SECRET=<__CHANGE_ME__Generated when you run `make auth`>
+# SWAGGER_UI_CLIENT_ID=<__CHANGE_ME__Generated when you run `make auth`>
 
 # The following 2 environment variables are only required
 # if you want to automated bundle registration.

